sharktank/tests/models/llama/sharded_llama_test.py

```diff
@@ -123,17 +228,19 @@ def make_decode_args(self, model: PagedLlamaModelV1) -> Dict[str, Any]:
         ).view(self.batch_size, -1)
         cache_state = model.cache.paged.allocate(page_count=self.cache_page_count)
         cache_state = [torch.rand_like(cache_state[0])]
-        return {
-            "tokens": decode_token_ids,
-            "attention_mask": attention_mask,
-            "start_positions": start_positions,
-            "seq_block_ids": seq_block_ids,
-            "cache_state": cache_state,
-        }
+        return OrderedDict(
+            [
+                ("tokens", decode_token_ids),
+                ("attention_mask", attention_mask),
+                ("start_positions", start_positions),
+                ("seq_block_ids", seq_block_ids),
+                ("cache_state", cache_state),
+            ]
+        )
 
     def make_equal_unsharded_and_sharded_decode_args(
         self, model: PagedLlamaModelV1, sharded_model: PagedLlamaModelV1
-    ):
+    ) -> Tuple[typing.OrderedDict[str, Any], typing.OrderedDict[str, Any]]:
```

Comment:
See comment above about importing `OrderedDict` from `typing` instead of `typing` itself